### PR TITLE
Update Fedora Remix Logo

### PR DESCRIPTION
**Pull request by crramirez**

### CODE REVIEW

**Comment by patrick330602** [comment]
Looks good, but I have a concern that for people importing fedora manually, if they use wslfetch, they will also see the logo. That is your suggestion on separating normal fedora and fedora remix?

**Comment by crramirez** [comment]
OK I will then modify the header to include the Remix check and differentiate

**Comment by patrick330602** [comment]
> OK I will then modify the header to include the Remix check and differentiate

Thanks for doing that

File: src/wslfetch.sh [hunk 1]
```diff
@@ -256,25 +256,25 @@ case "$distro" in
 			"${red}    \`/syyyyyyyyyyyyyyyyyyyyyyys+.    ${reset}"
 			"${red}      \`.:/+ooooooooooooooo+/:.\`      ${reset}");;
 	'oldfedora'|'fedora')
-		t="${light_blue}${bold}"
+		purple=$(echo -e '\e[38;5;140m')
```
Comment (patrick330602) [inline src/wslfetch.sh:259]: Also this should go in header

Comment (crramirez) [reply src/wslfetch.sh:259]: Yes but it modifies the original purple that you have. Maybe I can change the name?

**Comment by patrick330602** [comment]
Looks good to me. Merged.

**Pull request merged**